### PR TITLE
[daint-gpu dom-gpu] reverting magma and CP2K to use only cudatoolkit 

### PR DESCRIPTION
**Pull request by gppezzi**
(instead of craype-acel-nvidia60)

Builds were failing (most probably because of the variable `CRAY_TCMALLOC_MEMFS_FORCE=1`, but the issue needs to be confirmed)

### CODE REVIEW

File: easybuild/easyconfigs/c/CP2K/CP2K-6.1-CrayGNU-18.08-cuda-9.1.eb [hunk 1]
```diff
@@ -6,12 +6,6 @@ version = '6.1'
 cudaversion = '9.1'
 versionsuffix = '-cuda-%s' % cudaversion
 
-py_maj_ver = '2'
```
Comment (lucamar) [inline easybuild/easyconfigs/c/CP2K/CP2K-6.1-CrayGNU-18.08-cuda-9.1.eb]: We need to restore the `Python` variables in order to comply with the standard template described in the Wiki: I'll push a new version of the recipe for `CP2K`

**Comment by lucamar** [comment]
Thanks @gppezzi: the variable `CRAY_TCMALLOC_MEMFS_FORCE=1` itself does not create problems during the build and at runtime. However `craype-accel-nvidia60` loads as dependency `cray-libsci_acc`: if I unload the latter during the build, then at runtime `CP2K` runs correctly.
See a short summary here: `/scratch/snx3000/lucamar/cp2k/benchmark/legend.txt`
My tests show that having `cray-libsci_acc` only is not enough to reproduce the error, though.

Therefore I agree that for the time being it is safer to remove `craype-accel-nvidia60` from the dependencies, untile the issue gets clarified with the help of Cray experts (I will open a case).

**Comment by gppezzi** [comment]
I'm pasting the [benchmark results](https://gist.github.com/gppezzi/7c9fdf02e2f90c3b0580461cd502a654)  in a gist just for completeness

**Comment by lucamar** [comment]
retest this please

**Pull request merged**